src/components/date/date-test.stories.mdx

```diff
@@ -31,14 +32,40 @@ export const DateStory = (args) => {
       name="dateinput"
       value={state}
       onChange={setValue}
-      onBlur={(ev) => {action("onBlur")(ev.target.value)}}
+      onBlur={(ev) => {
+        action("onBlur")(ev.target.value);
+      }}
       onKeyDown={(ev) => action("onKeyDown")(ev.target.value)}
       onClick={(ev) => action("onClick")(ev.target.value)}
       {...getCommonTextboxArgsWithSpecialCaracters(args)}
     />
   );
 };
 
+export const NewValidationStory = (args) => {
```

Comment:
In Storybook, the validation message doesn't show by default nor there is a way to trigger it, is this intentional @samtjo? Would QA need this?

Reply:
Just use the error or warning control to type a message in:

![image](https://user-images.githubusercontent.com/14963680/155500052-e8e3cb78-1f5a-4641-a618-1a45016f7b4e.png)